app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(
+                f"{visitor["name"]}, you need "
+                f"to be vaccinated to visit the cafe!"
+            )
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date and expiration_date < date.today():
+            raise OutdatedVaccineError(
+                f"{visitor["name"]}, your vaccination has expired!"

```

Comment:
Same issue here. You're trying to use double quotes inside a string that is also enclosed in double quotes. This will raise a SyntaxError. To fix this, you can use single quotes for the string inside the f-string. For example: `f'{visitor['name']}, your vaccination has expired!'`